src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -103,6 +103,7 @@
             class="d-none mt-6"
             :maxSize="MAX_FILE_SIZE"
             :customErrorMessage.sync="customErrorMessage"
+            :isRequired="false"
```

Comment:
The file input component is hidden so we don't really use this, but it's more correct for the file input to not check if the file is required.